certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -3881,6 +3934,20 @@
         }
       }
     },
+    "ListTemplateDocumentsResponse": {
+      "type": "object",
+      "properties": {
+        "templateDocuments": {
+          "description": "Documents attached to a template.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EnvelopeDocument"
+          },
+          "x-ms-summary": "Template documents",
+          "x-ms-visibility": "advanced"
```

Comment:
why is the visibility advanced ?
I think you can delete this line.

Reply:
Yes, we dont need it. Deleted the line.